webdriver-spec.html

```diff
@@ -507,14 +507,14 @@
 <h2>Protocol</h2>
 
 <p>WebDriver <a>remote ends</a> must provide
- an HTTP-based, RESTish wire protocol
```

Comment:
On 2016-04-27 02:41, Andreas Tolfsen wrote:
> In webdriver-spec.html
> <https://github.com/w3c/webdriver/pull/300#discussion_r61230067>:
>
>> @@ -507,14 +507,14 @@
>>  <h2>Protocol</h2>
>>
>>  <p>WebDriver <a>remote ends</a> must provide
>> - an HTTP-based, RESTish wire protocol
>
> Why remove?

why use it? it's not really REST, and what's REST or not is a confusing 
debate anyway. the term "RESTish" does not convey any information and at 
best it is harmless. or it does what it did to me, getting readers to 
think "now what's that supposed to mean?"